File: Assignment 10/Problem4.c
//Accept N number from user and  accept Range, and Display Elements from that range

#include<stdio.h>
#include<stdlib.h>

void Range(int Arr[], int iLength, int iStart, int iEnd)
{
    int iCnt = 0;
  
    for(iCnt =0; iCnt<iLength ; iCnt++)
    {
            if(Arr[iCnt]>iStart && Arr[iCnt]<iEnd)
            {
                printf("%d\t",Arr[iCnt]);
            }
    }
	
}

int main()
{
    int iSize = 0, iCnt = 0, iNo1 = 0,iNo2=0;
    int *ptr = NULL;   
    int iRet=0;

    printf("Enter number of elements\n");
    scanf("%d",&iSize);

    ptr = (int *)malloc(sizeof(int) * iSize);

    printf("Enter the values\n");
    for(iCnt = 0; iCnt < iSize; iCnt++)
    {
        scanf("%d",&ptr[iCnt]);
    }

    printf("Enter the Starting point \n");
    scanf("%d",&iNo1);
	
    printf("Enter the End point \n");
    scanf("%d",&iNo2);

    Range(ptr, iSize, iNo1,iNo2);
	
    return 0;
}

File: Assignment 9/Problem4.java
//Accept N number from user and return frequency of 11 from it.

import java.lang.*;
import java.util.*;

class Numbers
{
	public int Frequency(int Brr[])
	{
		int iFrequency =0;

		
		int iDFrequency =0;
		
		for(int iCnt=0; iCnt<Brr.length;iCnt++)
		{
			if(Brr[iCnt] == 11)  
			{
				iFrequency++;
			}
		}
		
		
		return iFrequency;
	}
}

class Problem4
{
	
	public static void main(String a[])
	{
		Numbers nobj = new Numbers();
		Scanner sobj =new Scanner(System.in);
		
		System.out.println("Enter Number of Elements");
		int iSize = sobj.nextInt();
		
		int Arr[] = new int[iSize];
		
		System.out.println("Enter Elements of Array");
		
		for(int iCnt=0;iCnt<iSize;iCnt++)
		{
			Arr[iCnt] = sobj.nextInt();	
		}
		
		int iRet = nobj.Frequency(Arr);
		
		System.out.println("Frequency of 11 is = "+iRet);
		
	}
}